internal/pkg/pluginengine/getOutputs_test.go

```diff
@@ -0,0 +1,89 @@
+package pluginengine
```

Comment:
We should use the ginkgo framework for the ut.

Reply:
will do it in an other PR for refactor